lib/gooddata/helpers/data_helper.rb

```diff
@@ -7,6 +7,7 @@
 require 'csv'
 require 'digest'
 require 'open-uri'
+require_relative '../cloud_resources/blobstorage/blobstorage_client'
```

Comment:
should load blob storage api only when user defined blob storage input source

Reply:
Fixed, Moved require_relative into blobStorage